projects/ion/src/lib/popconfirm/popconfirm.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, Input } from '@angular/core';
+import { Subject } from 'rxjs';
+import { StatusType } from '../core/types';
+
+export interface PopConfirmProps {
+  ionPopConfirmTitle: string;
+  ionPopConfirmType?: StatusType;
+  ionPopConfirmDesc?: string;
+  ionConfirmText?: string;
+  ionCancelText?: string;
+}
+
+@Component({
+  selector: 'ion-popconfirm',
+  templateUrl: './popconfirm.component.html',
+  styleUrls: ['./popconfirm.component.scss'],
+  exportAs: 'PopConfirmComponent',
+})
+export class PopConfirmComponent {
+  @Input() ionPopConfirmTitle: string;
+  @Input() ionPopConfirmDesc: string;
```

Comment:
```suggestion
  @Input() ionPopConfirmDescription: string;
```
Acredito que não seja uma boa prática abreviar